csharp/Svix/SvixOptions.cs

```diff
@@ -2,17 +2,24 @@ namespace Svix
 {
     public class SvixOptions
     {
+        [Obsolete("BaseUrl is deprecated, please use ServerUrl instead.")]
         public string BaseUrl { get; }
+        public string ServerUrl { get; }
         public List<int> RetryScheduleMilliseconds { get; } = [50, 100, 200];
         public int TimeoutMilliseconds { get; } = 15000;
 
+        /// <param name="serverUrl">The server URL to connect to.</param>
+        /// <param name="baseUrl">[Deprecated] Please use serverUrl parameter instead.</param>
+#pragma warning disable CS0618
         public SvixOptions(
-            string baseUrl,
+            string serverUrl,
             int timeoutMilliseconds = 15000,
-            List<int>? retryScheduleMilliseconds = null
+            List<int>? retryScheduleMilliseconds = null,
+            string? baseUrl = null
         )
         {
-            BaseUrl = baseUrl;
+            ServerUrl = baseUrl ?? serverUrl;
+            BaseUrl = baseUrl ?? serverUrl;
```

Comment:
Why are we setting both? The structure should only have one variable and this gets both.
Additionally, `serverUrl` should be the priority, not `baseUrl`...